soc/arm/atmel_sam0/common/soc_samd5x.c

```diff
@@ -116,8 +117,14 @@ static int atmel_samd_init(void)
 		dfll_div = 1;
 	}
 
-	/* enable the Cortex M Cache Controller */
-	CMCC->CTRL.bit.CEN = 1;
+	/*
+	 * Force Cortex M Cache Controller disabled
+	 *
+	 * It is not clear if regular Cortex-M instructions can be used to
+	 * perform cache maintenance or this is a proprietary cache controller
+	 * that require special SoC support.
+	 */
+	CMCC->CTRL.bit.CEN = 0;
```

Comment:
Please break this into a dedicated commit.

Reply:
Done!